deploy/crds/devopsconsole_v1alpha1_component_crd.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: components.devopsconsole.openshift.io
+spec:
+  group: devopsconsole.openshift.io
+  names:
+    kind: Component
+    listKind: ComponentList
+    plural: components
+    singular: component
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            buildType:
+              type: string
```

Comment:
Since we would have a limited number of `buildTypes`, I think we should add the enum here. See https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/quickstart/olm.yaml#L1011 for an example.

What do you think @corinnekrych

Reply:
5046adb
for now only added nodejs, more will be added as we progress

Reply:
Sounds good :+1: